bin/tennicam_client_display.py

```diff
@@ -21,28 +29,65 @@
 TENNICAM_CLIENT_DEFAULT_SEGMENT_ID = "tennicam_client"
 
 
-def _get_handle():
+def _get_vicon_table_pose(vicon_segment_id: str) -> Transformation:
+    # only import if actually needed
+    import pam_vicon_o80
```

Comment:
why the import here and not at the top ?

Reply:
I thought it might be nice to keep the dependency on Vicon-related packages optional (i.e. you can still run the script without the package being present as long as you don't set the `--vicon` argument).